### PR TITLE
No issue - Inject excluded tables configuration into the backupper factory instead of the backup command

### PR DESCRIPTION
**Pull request by Lonnytunes**

### CODE REVIEW

File: src/Backupper/BackupperFactory.php [hunk 1]
```diff
@@ -58,6 +96,9 @@ public function create(?string $connectionName = null): AbstractBackupper
 
         \assert($backupper instanceof AbstractBackupper);
 
+        if (isset($this->excludedTables[$connectionName])) {
```
Comment (SimonMellerin) [inline src/Backupper/BackupperFactory.php:99]: Don't know why, but I prefer `\key_exists($connectionName, $this->excludedTables)`

Do you know if one is better than the other?

File: src/Backupper/AbstractBackupper.php [hunk 2]
```diff
@@ -78,7 +78,18 @@ public function getDestination(): ?string
 
     public function setExcludedTables(array $excludedTables): self
     {
-        $this->excludedTables = $excludedTables;
+        foreach ($excludedTables as $table) {
```
Comment (SimonMellerin) [inline src/Backupper/AbstractBackupper.php:81]: I think you should reset `$this->excludedTables` here.

Or the method name should be `::addExcludedTables`. But, if we consider how you use this method in `BackupCommand`, I think it should definitively be a `setExcludedTable` with a reset.

```suggestion
        $this->excludedTables = [];
        foreach ($excludedTables as $table) {
```

Comment (SimonMellerin) [reply src/Backupper/AbstractBackupper.php:81]: > If given, overrides excluded tables parameter from bundle configuration.

From the command description for the `excluded-table` option

**Review by SimonMellerin — CHANGES_REQUESTED** [review]

File: src/Backupper/AbstractBackupper.php (hunk 2 above)
Comment (Lonnytunes) [reply src/Backupper/AbstractBackupper.php:82]: > I think you should reset $this->excludedTables here.

Indeed! I applied your suggestion :pray:

**Review by SimonMellerin — APPROVED** [review]

**Pull request merged**